src/vm/jitinterfacegen.cpp

```diff
@@ -221,10 +221,12 @@ void InitJITHelpers1()
 #endif // _DEBUG
         ))
     {
-
         // if (multi-proc || server GC)
         if (GCHeap::UseAllocationContexts())
```

Comment:
Could you please fix the UseAllocationContexts implementation in gc.h to always use allocation contexts for FEATURE_PAL so that we do not need to worry about optimizing the non-allocation context path?